### PR TITLE
update to password specifics

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** updated info on how passwords are verified
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-518496](https://oktainc.atlassian.net/browse/OKTA-518496)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/users/index.md [hunk 1]
```diff
@@ -4095,15 +4095,16 @@ When a user has a valid password, or imported hashed password, or password hook,
...
 
-- Must be a minimum of 8 characters
+- Must be a minimum of eight characters
 - Must have a character from the following groups:
   - Upper case
   - Lower case
   - Digit
-- Must not contain the user's login or parts of the the login when split on the following characters: `,` `.` `_` `#` `@`
-  - *For example, a user with login `isaac.brock@example.com` will not be able set password brockR0cks! as the password contains the login part `brock`.*
+- Must not contain the user's sign-in ID or parts of the sign-in ID when split on the following characters: `,`, `.`, `_`, `#`, `@`,`-`. Okta only considers the parts of the sign-in ID that contain at least four characters. Additionally, the parent domain (such as `.com` in the following example) isn't considered.<br>
```
[the remark this answers is not in the document]
Comment (shamil-okta) [reply packages/@okta/vuepress-site/docs/reference/api/users/index.md]: @susanharper-okta

Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/users/index.md:4103]:
```suggestion
- Must not contain the user's sign-in ID or parts of the sign-in ID when split on the following characters: `,`, `.`, `_`, `#`, `@`, `-`. Okta only considers the parts of the sign-in ID that contain at least four characters. Additionally, the parent domain (such as `com` in the following example) isn't considered.<br>
```

**Review by susanharper-okta — COMMENTED** [review]
Addressed Shamil's review that didn't show up in here for some reason.

**Review by shamil-okta — APPROVED** [review]
LGTM

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

**Pull request merged**